pyflakes/test/test_imports.py

```diff
@@ -1093,6 +1093,15 @@ def test_concatenationAssignment(self):
         __all__ = ['a'] + ['b'] + ['c']
         ''', m.UndefinedExport, m.UndefinedExport, m.UndefinedExport, m.UnusedImport)
 
+    def test_tupleConcatenationAssignment(self):
```

Comment:
the new tests are suggested to use `snake_case` (the camel case ones are the older code)

Reply:
Done